data-access/Models/ArtefactModels.cs

```diff
@@ -7,12 +7,16 @@ public class ArtefactModel
 		public string Name { get; set; }
 		public string Mask { get; set; }
 		public uint MaskEditorId { get; set; }
-		public string TransformMatrix { get; set; }
-		public uint TransformMatrixEditorId { get; set; }
+		public float? Scale { get; set; }
+		public float? Rotate { get; set; }
+		public uint? TranslateX { get; set; }
+		public uint? TranslateY { get; set; }
```

Comment:
Also, obviously in the database the X and Y values are stored in separate fields. In the model, however, perhaps we can group them in an Offset structure (that has X and Y properties)? For instance, there's no case where TranslateX is null and TranslateY has a value - they're both null or both not null.

Reply:
I like the sound of that.  I will leave this as is in the database model (it is used directly by Dapper to serialize the DB query), but in the corresponding DTO I will serialize it to a translate or offset object with x and y properties.  Thanks.